llarp/path/path.cpp

```diff
@@ -793,7 +798,7 @@ namespace llarp
         }
         LogInfo(Name(), " ", Endpoint(), " Rejected exit");
         MarkActive(r->Now());
-        return InformExitResult(msg.B);
+        return InformExitResult(Time_t(msg.B));
```

Comment:
`Time_t` ? do we want to get rid of `Time_t`?

Reply:
it's gone now